python/cudf/cudf/tests/test_options.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import pytest
+
+import cudf
+
+
+@pytest.fixture(scope="module")
+def odd_option():
+    cudf.options._register_option(
+        "odd_config",
+        1,
+        "An odd option.",
+        lambda x: x % 2 == 1,
+    )
+    yield
+    cudf.options._CUDF_OPTIONS.pop("odd_config")
```

Comment:
This should use `del` because it doesn't use the value returned by `pop`.
```suggestion
    del cudf.options._CUDF_OPTIONS["odd_config"]
```